src/types/index.ts

```diff
@@ -0,0 +1,75 @@
+import type { GovUKPayment } from "./gov-uk-payment";
+import type { Breadcrumbs } from "./breadcrumbs";
+
+export * from "./data";
+export * from "./component";
+export * from "./gov-uk-payment";
+export * from "./flow";
+export * from "./sections";
+export * from "./breadcrumbs";
+export * from "./component";
+export * from "./payment-request";
+
```

Comment:
There are still some types that could be organized below but I've left them for future updates once sensible groupings emerge (feel free to update these if you see a grouping that makes sense).